website/docs/tutorials/working-with-files.md

```diff
@@ -27,12 +27,16 @@ the fetched file as a blob. It is then up to you to decode it appropriately.
 ```typescript
 import { unstable_fetchFile } from "lit-solid";
 
-const response = await unstable_fetchFile(
+const file = await unstable_fetchFile(
   "https://example.com/some/interesting/file"
 );
-if (response.ok) {
-  const myFile = await response.blob();
-}
+// file is a Blob (see https://developer.mozilla.org/en-US/docs/Web/API/Blob)
+console.log(
+  `Fetched a ${file.resourceInfo.contentType} file from ${file.resourceInfo.fetchedFrom}.`
```

Comment:
If we want to be able to also include this data in the dataset (I don't see how, since it might not be a dataset, but anyway... DO IT IN RDF!), we should stimulate the use the functions we expose here:

```suggestion
  `Fetched a ${getContentType(file)} file from ${file.resourceInfo.fetchedFrom}.`
```

Likewise for `isLitDataset`, and I suppose we might want to add a function to expose a Resource's original URL as well?

Reply:
Good point. A `getFetchedFrom` is added in 0cfd4f4b1791538047dbf776f021825f07906fca, and the doc is updated in  40ddcc7addb30de2306bf2b88e311f3b5f56feba.